updating/understanding-openshift-updates.adoc

```diff
@@ -17,6 +17,13 @@ After the CVO receives the update image from the registry, the CVO then applies
 Operators previously installed through Operator Lifecycle Manager (OLM) follow a different process for updates. See xref:../operators/admin/olm-upgrading-operators.adoc#olm-upgrading-operators[Updating installed Operators] for more information.
 ====
 
+include::modules/update-availability.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
+
+* xref:../updating/understanding-upgrade-channels-release.adoc#understanding-upgrade-channels-releases[Understanding update channels and releases]
```

Comment:
Should this be called "understanding update channels..." vs. "understanding upgrade channels..." on the left outline/table of contents?

Reply:
That's an inconsistency that needs to be fixed, but it's the TOC entry that needs to be changed to "update" instead of "upgrade". I can do that in a separate PR.